test/fixtures/nested-inside-other-func.css

```diff
@@ -1,3 +1,54 @@
+:root {
+	--some-color: red;
+	--some-width: 150px;
+	--some-other-width: 50px;
+	--some-margin: 25px;
+	--some-padding: 20px;
+}
+
+.box-foo {
+	background-color: color(var(--some-color));
+}
+
+.box-foo {
+	background-color: color(var(--missing, white));
+}
```

Comment:
Let's split these cases out to separate fixture files so the output is easier to understand when it fails

 - `nested-inside-other-func-with-fallback`
 - `nested-inside-calc-func`
 - `nested-inside-calc-func-with-fallback`
 - `nested-inside-calc-func-with-fallback-var`
 - etc

Reply:
Done. Added more use-case tests